moulinette/actionsmap.py

```diff
@@ -511,8 +517,13 @@ def generate_cache(klass, namespaces=None):
             with open(am_file, 'r') as f:
                 actionsmaps[n] = yaml.load(f)
 
+            # clean old cached files
+            for i in os.listdir('%s/actionsmap/' % pkg.cachedir):
+                if i.endswith(".pkl"):
```

Comment:
I think [`glob.glob`](https://docs.python.org/2/library/glob.html#glob.glob) could be better and more efficient for that, isn't it?

For example:
```python
for path in glob.iglob('%s/actionsmap/*.pkl' % pkg.cachedir):
    os.remove(path)
```